packages/analytics/src/rudderstack.ts

```diff
@@ -155,7 +209,10 @@ export class RudderStack {
      */
     track<T extends keyof TEvents>(event: T, payload: TEvents[T]) {
         if (this.has_initialized && this.has_identified) {
-            RudderAnalytics.track(event, payload);
+            RudderAnalytics.track(event, {
+                ...payload,
+                account_type: this.account_type,
```

Comment:
taken from https://github.com/binary-com/deriv-app/pull/9331/files since we don't know which PR will be merged sooner.